internal/storage/firestore/firestore.go

```diff
@@ -110,3 +109,7 @@ func IterateKills(iter *firestore.DocumentIterator) ([]*storage.Kill, error) {
 	}
 	return kills, nil
 }
+
+func (s *KillStore) CloseKillStore() {
+	defer s.client.Close()
```

Comment:
defer isn't required here as it will do the same thing as just calling the method